wsi_deid/import_export.py

```diff
@@ -151,10 +149,26 @@ def readExcelFiles(filelist, ctx):
                     ctx.update(message=message)
                     logger.info(message)
                 totalErrors.append({'name': name, 'errors': errors})
-            if not name:
-                continue
             count += 1
-            if name not in manifest or (timestamp > manifest[name]['timestamp'] and not errors):
+            if not name:
+                if (not errors and
+                   ('ScannedFileName' not in properties and 'InputFileName' not in properties)):
```

Comment:
We probably want to change this to check if the property is missing or empty for these, so that you could have *some* rows with an input file name).